src/configure/templateInputHelper/InputControlProvider.ts

```diff
@@ -87,21 +102,22 @@ export class InputControlProvider {
             return;
         }
         var dependentInputControlArray = this._getInputDependencyArray(inputControl, [inputDes.defaultValue], false);
+        var dependentClientInputMap = this._getClientDependencyMap(inputControl, [inputDes.defaultValue]);
 
-        var defaultValue = this._computeMustacheValue(inputDes.defaultValue, dependentInputControlArray);
+        var defaultValue = this._computeMustacheValue(inputDes.defaultValue, dependentInputControlArray, dependentClientInputMap);
         if (defaultValue !== inputControl.getValue()) {
             inputControl.setValue(defaultValue);
         }
     }
 
 
-    private _computeMustacheValue(mustacheExpression: string, dependentInputControlArray: InputControl[]): string {
+    private _computeMustacheValue(mustacheExpression: string, dependentInputControlArray: InputControl[], dependentClientInputMap: StringMap<any>): string {
 
         var dependentInputValues = this._getInputParameterValueIfAllSet(dependentInputControlArray);
         if (dependentInputControlArray && dependentInputControlArray.length > 0 && !dependentInputValues) {
             return "";
         } else {
-            return MustacheHelper.render(mustacheExpression, { inputs: dependentInputValues });
+            return MustacheHelper.render(mustacheExpression, { inputs: dependentInputValues, client: dependentClientInputMap });
```

Comment:
can you please figure out what it needs to be present in system context as system context is also being used  in templates.

Reply:
As discussed, will be handling later while working with yml files